lnrpc/rpc.proto

```diff
@@ -1734,6 +1734,12 @@ message ListInvoiceRequest {
 
     /// The max number of invoices to return in the response to this query.
     uint32 num_max_invoices = 5 [json_name = "num_max_invoices"];
+
+    /**
+    If set, the invoices returned will result from seeking backwards from the
+    specified index offset.
+    */
+    bool reversed = 6 [json_name = "reversed"];
```

Comment:
Not sure if we should leave this as `6`. There were some other fields that were removed when the `listinvoices` PR was merged and the ones above were not modified to reflect that.

Reply:
Can you point to the commit that removed those fields? Generally one should not reuse field tags, as long as they are added we should be okay :)

Reply:
The removed fields won't show as they weren't committed yet, but this is the commit where they were added. 

https://github.com/lightningnetwork/lnd/pull/1763/commits/1c7f10a467952e0d0977847db80577f3315c6ac8#diff-f6ec8c0f52caccfaec5656028d316c7fR1720